docs/testing/raid-test-plan.md

```
@@ -0,0 +1,1281 @@
+  echo 1 > /proc/sys/dev/raid/speed_limit_min
+'
+```
+
+Create raid5 + PVC. Immediately check:
+
+```bash
+oc exec -n $NS $VG_POD -- curl -s http://localhost:8080/metrics | grep lvms_raid_sync
+```
+
+**Expected:**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)